modin/pandas/series.py

```diff
@@ -543,7 +543,11 @@ def values(self):  # noqa: RT01, D200
         """
         Return Series as ndarray or ndarray-like depending on the dtype.
         """
-        return super(Series, self).to_numpy().flatten()
+
+        def values(df):
+            return df.values
+
+        return self._default_to_pandas(values)
```

Comment:
```suggestion
        return self.to_numpy().flatten()
```

Reply:
Done. I also removed the flatten() since presumably Series will already be 1-D, and flatten() is not necessary. Let me know if this is not a safe assumption.